### PR TITLE
01_FW

### PR DESCRIPTION
**Pull request by Rjeey-dev**

### CODE REVIEW

File: vabashyn/01_FW/zadanie2.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            $("#passw1, #passw2").on("keyup", function(){
+                var psw1 = $("#passw1").val();
+                var psw2 = $("#passw2").val();
+                if (psw1 != psw2) {
+                    $("#error").text("Введенные вами пароли не совпадают").css('color', 'red');
+                    $("#passw1").css('background-color', '#FF0000');
+                    $("#passw2").css('background-color', '#FF0000');
+                } else {
+                    $("#error").text("Ваши пароли совпадают").css('color', 'black');
+                    $("#passw1").css('background-color', '#7CFC00');
```
Comment (pluhin) [inline vabashyn/01_FW/zadanie2.html]: You can use "#passw1, #passw2" selector

File:  [hunk 2]
```diff
@@ -0,0 +1,101 @@
+[33mcommit 1534f7c58068cfc09e0772cd884f3d80c1bdcca3[m[33m ([m[1;36mHEAD -> [m[1;32mm-sc1-60-19[m[33m, [m[1;31morigin/m-sc1-60-19[m[33m)[m
```
Comment (pluhin) [inline ]: What is it? Could you please remove

**Review by pluhin — CHANGES_REQUESTED** [review]

File:  (hunk 2 above)
Comment (pluhin) [inline :1]: What is the file here?

File: vabashyn/04_CSS/zadanie3.html [hunk 3]
```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
```
Comment (pluhin) [inline vabashyn/04_CSS/zadanie3.html:2]: We use html5

**Review by pluhin — CHANGES_REQUESTED** [review]

File: vabashyn/04_CSS/zadanie3.html (hunk 3 above)
Comment (Rjeey-dev) [reply vabashyn/04_CSS/zadanie3.html:2]: Sorry but I do not understand what this file is.My repository does not have this file.

Comment (pluhin) [reply vabashyn/04_CSS/zadanie3.html:2]: Could you please remove  xmlns="http://www.w3.org/1999/xhtml"

**Pull request closed without merging**